packages/fluentui/react-northstar/src/utils/whatInput.ts

```diff
@@ -108,6 +108,20 @@ const addListeners = (eventTarget: Window) => {
   eventTarget.addEventListener('keyup', eventBuffer, true);
 };
 
+const ignoreKeyboardFocused = (document: Document, eventKey: number) => {
```

Comment:
I would pick a clearer name for this: `keyboardInputFocused` ?

Also the function definitely needs some docstring to explain what is going on and why this is the case